### PR TITLE
Update skyview with extended mode S data fields

### PR DESCRIPTION
**Pull request by drakeapps**

### CODE REVIEW

File: public_html/formatter.js [hunk 1]
```diff
@@ -156,6 +168,16 @@ function format_distance_long(dist, displayUnits, fixed) {
 	return dist_text;
 }
 
+function format_distace_short (dist, displayUnits) {
```
Comment (mutability) [inline public_html/formatter.js]: typo for "distance"

File: public_html/formatter.js [hunk 2]
```diff
@@ -167,6 +189,15 @@ function convert_distance(dist, displayUnits) {
 	return (dist / 1852); // meters to nautical miles
 }
 
+// dist in meters
+// converts meters to feet or just returns meters
+function convert_distance_short(dist, displayUnits) {
+	if (displayUnits === "imperial") {
+		return (dist * 3.28084); // meters to feet
```
Comment (mutability) [inline public_html/formatter.js]: Prefer `dist / 0.3048` which is exact.

File: public_html/planeObject.js [hunk 3]
```diff
@@ -28,7 +28,14 @@ function PlaneObject(icao) {
         this.nav_altitude   = null;
         this.nav_heading    = null;
         this.nav_modes      = null;
-        this.nav_qnh        = null;
+		this.nav_qnh        = null;
```
Comment (mutability) [inline public_html/planeObject.js]: I know the tab/space stuff is all over the place in this code, but can we keep it to spaces here to be consistent with the surrounding stuff?

File: public_html/style.css [hunk 4]
```diff
@@ -471,12 +508,12 @@ select.error, textarea.error, input.error {
 
 .infoRowTitle {
     display: inline-block;
-    width: 50%;
+    /* width: 50%; */
```
Comment (mutability) [inline public_html/style.css]: Remove the dead CSS?

File: public_html/style.css [hunk 5]
```diff
@@ -471,12 +508,12 @@ select.error, textarea.error, input.error {
...
 }
 
 .infoRowContent {
     display: inline-block;
-    width: 40%;
+    /* width: 40%; */
```
Comment (mutability) [inline public_html/style.css]: More dead CSS

File: public_html/style.css [hunk 6]
```diff
@@ -517,7 +561,7 @@ select.error, textarea.error, input.error {
 .lightGreyBackground {
     background-color: #efefef;
 }
-
+/*
```
Comment (mutability) [inline public_html/style.css]: More dead CSS

**Comment by drakeapps** [comment]
Cleaned up css/spacing and typo.

**Pull request merged**